run_array_analyzer.py

```diff
@@ -131,6 +131,16 @@ def workflow(input_folder_, output_folder_, debug=False):
     # parsing .xml
     fiduc, spots, repl, params = create_xml_dict(xml_path)
 
+    # add fiducial locations and pixel size to params
+    # TODO: ged fiducial locations from xkappa-biotin locations in xml file
+
+    # for scienion images
+    # params['fiducial_locations'] = [(0, 0), (0, 1), (0, 5), (7, 0), (7, 5)]
+    # params['pixel_size'] = 0.0049 # in mm
+    # for octopi images
+    params['fiducial_locations'] = [(0, 0), (0, 5), (5, 0), (5, 5)]
+    params['pixel_size'] = 0.00185 # in mm
```

Comment:
Why not create two constants at the top of the file, e.g. 
PIXEL_SIZE_SCENION
PIXEL_SIZE_OCTOPI
then add a command line argument that can be either scenion or octopi?

Reply:
I think this makes sense.

Additionally, the fiducial locations are actually present in the .xml.  I don't call the function to parse it because, until now, we didn't use them.  This could be automatically included in a later PR